multiqc/modules/hicpro/hicpro.py

```diff
@@ -169,7 +186,7 @@ def parse_hicpro_stats(self, f, rsection):
                         self.hicpro_data[s_name][s[0]] = float(s[1])
                     # Otherwise just store the value as is.
                     except ValueError:
-                        self.hicpro_data[s_name][s[0]] = s[1]
+                        log.error(f"Could not parse value for key '{s[0]}' as a number in sample '{s_name}': '{s[1]}'")
```

Comment:
Do we want to keep the value even if it's not a number?

Reply:
I was thinking that they are attempted to be parsed as numbers for some calculations further in the code, so there is no reason to keep strings. But perhaps there are other metrics that are not used for calculations and we want to keep, so I'll revert and test this bit

Reply:
Reverted that, and did more debugging in hicpro, there were actually a few issues with it:
- the search patterns didn't account for half of input test data (e.g. `.mmapstat` was incouded by `.mapstat` was missing; `_assplit.stat` was missing, etc)
- there were warning about missing data on plots, just needed to add a check that the required data was found like in other modules
- improved debug info about duplicated key for a sample

The PR is heavy in single module, perhaps should split it up for clarity 🤔

Reply:
Moving this stuff into a separate PR